### ISSUE TITLE
\anon to hide content parts in "anonymous" mode

### ISSUE THREAD

**Issue by yegor256** [borisveytsman/acmart#455]
What do you think about this command?

```tex
\makeatletter\newcommand\anon[2][ANONYMIZED]{%
  \if@ACM@anonymous%
    {\color{gray}#1}%
  \else%
    #2%
  \fi%
}\makeatother
```

This may be useful when a paper is being prepared for double-blinded review. Some parts of the content may need to be anonymized, for example:

```
The work was done by \anon{Foo University}.
```

**Comment by yegor256** [comment]
If you think this command would make sense, I can submit a pull request

**Comment by borisveytsman** [comment]
Thanks, this could work.   I am not sure we need color here, however

**Comment by borisveytsman** [comment]
Closed by #456

**Comment by a-a-k** [comment]
@yegor256 Thank you for this indispensable feature — and greetings from Innopolis!
